cmd/kpromo/cmd/pr/pr.go

```diff
@@ -316,33 +355,35 @@ func runPromote(opts *promoteOptions) error {
 	}
 
 	// Push to fork
-	if mustRun(opts, fmt.Sprintf("Push changes to user's fork at %s/%s?", userForkOrg, userForkRepo)) {
-		logrus.Infof("Pushing manifest changes to %s/%s", userForkOrg, userForkRepo)
+	if mustRun(opts, fmt.Sprintf("Push changes to user's fork at %s/%s?", opts.userFork, k8sioRepo)) {
+		logrus.Infof("Pushing manifest changes to %s/%s", opts.userFork, k8sioRepo)
 		if err := repo.PushToRemote(github.UserForkName, branchname); err != nil {
-			return fmt.Errorf("pushing %s to %s/%s: %w", github.UserForkName, userForkOrg, userForkRepo, err)
+			return fmt.Errorf("pushing %s to %s/%s: %w", github.UserForkName, opts.userFork, k8sioRepo, err)
```

Comment:
This looks wrong to me, why did you replace `userForkOrg` with `opts.userFork` and `userForkRepo` with `k8sioRepo`?